src/Symfony/Component/Notifier/Bridge/Mercure/README.md

```diff
@@ -0,0 +1,23 @@
+Mercure Notifier
+===================
+
+Provides [Mercure](https://github.com/symfony/mercure) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```ini
```

Comment:
Sorry please remove the `ini`, I was to fast, we decided to not do that, thanks!